lib/split/algorithms/weighted_deterministic.rb

```diff
@@ -0,0 +1,20 @@
+require 'digest/murmurhash'
+module Split
+  module Algorithms
+    module WeightedDeterministic
+      def self.choose_alternative(experiment, split_id)
+        hash = Digest::MurmurHash1.rawdigest("#{experiment.key}:#{split_id}")
```

Comment:
think we can use more recent version as suggested by Austin Appleby: https://github.com/aappleby/smhasher/wiki/MurmurHash